docs/pages/project/contributing/contributing-docs.md

```diff
@@ -186,6 +188,14 @@ Inline formatting:
 
 \`code snippet\`: `code snippet`
 
+Language specific:
+
+```
+```(language name)
+  code snippet
+```
+```
```

Comment:
Also could you check here once, only the top back-ticks are available on the deploy preview, https://6109775ef5c2023f660c5b4a--meshery-docs-preview.netlify.app/project/contributing-docs#using-the-features-of-meshery-docs